_layouts/exit-who.html

```diff
@@ -0,0 +1,89 @@
+{% include header.html %}
+  <main id="main-content" class="grid-container">
+    {% include step-indicator.html %}
+    {{ content }}
+    <div class="usa-accordion">
+  
+      <!-- Use the accurate heading level to maintain the document outline -->
+      <h4 class="usa-accordion__heading">
+        <button class="usa-accordion__button"
+          aria-expanded="true"
+          aria-controls="a1">
+          Law Enforcement Agency
```

Comment:
We should separate the content out from this template. I can help with this if needed. Or you can look at the FAQ page template. I did something similar there.